tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1360,4 +1360,39 @@ public function commentsProvider() : array
             'valid with extra opening brackets' => ['(DC2Type:should((.stop)).before)', 'should((.stop', $currentType],
         ];
     }
+
+    /**
+     * Returns literals that are platform specific escaped.
+     *
+     * @return array
+     */
+    protected function getEscapedLiterals() : array
+    {
+        return [
+            "\\'", // A single quote
+            '\\"', // A double quote
+            '\\n', // A new-line character
+            '\\r', // A carriage return character
+        ];
+    }
+
+    public function testEscapedDefaultValueMustBePreserved() : void
+    {
+        $value = implode('+', $this->getEscapedLiterals());
+
+        $table = new Table('string_escaped_default_value');
+        $table->addColumn('def_string', 'string', array('default' => $value));
+        $table->addColumn('def_foo', 'string');
```

Comment:
This column doesn't seem needed for the test. Inserting an empty array should work.

Reply:
Unfortunately inserting an empty array does not work, at the very least on PostgreSQL and possibly on other platforms. While this most likely should be fixed, I think it is out of the scope of this PR.